Formula/kustomize.rb

```diff
@@ -0,0 +1,31 @@
+class Kustomize < Formula
+  desc "Template-free customization of Kubernetes YAML manifests"
+  homepage "https://github.com/kubernetes-sigs/kustomize"
+  url "https://github.com/kubernetes-sigs/kustomize.git",
+      :tag => "v1.0.6",
+      :revision => "017c4ae0aa19195db2a51ecc5aa82c56a1f1c99b"
+  head "https://github.com/kubernetes-sigs/kustomize.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "0"
```

Comment:
Isn't it off by default anyway? Maybe I'm misremembering.

Reply:
No. This is required to enable static compilation when the OS you're building on is the same as the target OS (GOOS). In this case both are `darwin`, so this is needed.

Reply:
Super.